apps/k8s-io/configmap-nginx.yaml

```diff
@@ -346,8 +346,25 @@ data:
         listen 80;
 
         location / {
-          rewrite ^/$                  https://github.com/kubernetes/kubernetes/releases redirect;
-          rewrite ^/([^/]*)(/.*)?$     https://github.com/kubernetes/kubernetes/tree/$1$2 redirect;
+          # Special case for versions up to v1.33 - redirect to bit.ly
+          if ($uri ~ "^/v([0-9])([0-9]+)/([a-zA-Z]+)$") {
+            set $major $1;
+            set $minor $2;
+            set $remaining $3;
+
+            # For versions <=1.33, redirect to bit.ly
+            if ($major = "1" && $minor <= "33") {
+              return 301 https://bit.ly/k8s$major$minor-$remaining;
+            }
```

Comment:
Thinking about this again, this could allow for something similar to subdomain takeover or unknown redirect.

`rel.k8s.io/v123/steal-your-passwords` would redirect externally to `https://bit.ly/k8s123-steal-your-passwords` which isn't safe imo. 

For maintaining backwards compatibility, we can have a generic links.md in sig-release which would have the generic URLs for previous release.

Reply:
We should also constrain the lower bound if we go this route.

I think it might be best to directly include them here without the bit.ly link in the middle?

Reply:
Yeah, having bit ly remains doesn't seem like a great solution. I can write a short script which can auto populate links.md for the older releases. 

Updated this to do a redirect to sig-release only.